src/OrchardCore.Modules/OrchardCore.OpenId/Configuration/OpenIdServerConfiguration.cs

```diff
@@ -182,13 +184,28 @@ public void Configure(string name, JwtBearerOptions options)
 
         public void Configure(string name, OpenIddictValidationOptions options)
         {
+            if (SupportedValidationSchemes().Any(scheme => scheme.Equals(name)))
```

Comment:
This class shouldn't amend the validation options of the main handler, only the private userinfo scheme (that will disappear once migrating to OpenIddict 3.0). Consider removing that and adding the same logic in `OpenIdValidationConfiguration`.

Reply:
@kevinchalet the idea was to keep this logic in one place as I have to use the construction ` CreateTenantScope(settings.Tenant).UsingAsync` to resolve ServerSettings options. But I got what you mean and I'll push up the changes with all your suggestions. Suck a bit with rebasing :)